iocage_lib/ioc_common.py

```diff
@@ -1032,3 +1032,32 @@ def parse_package_name(pkg):
         'revision': revision,
         'epoch': epoch,
     }
+
+
+def get_host_gateways():
+    netifaces_af_mapping = {
+        'Internet': netifaces.AF_INET,
+        'Internet6': netifaces.AF_INET6
+    }
+
+    output = su.run(['netstat', '-r', '-n', '--libxo', 'json'],
+                    stdout=su.PIPE)
+    route_families = (json.loads(output.stdout)
+                      ['statistics']
+                      ['route-information']
+                      ['route-table']
+                      ['rt-family'])
+
+    gateways = {'default': {}}
+    for af in ('Internet', 'Internet6'):
+        route_entries = list(filter(
+            lambda x: x['address-family'] == af, route_families)
+        )[0]['rt-entry']
+        default_route = list(filter(
+            lambda x: x['destination'] == 'default', route_entries)
+        )
+        if default_route and 'gateway' in default_route[0]:
+            gateways['default'][netifaces_af_mapping[af]] = (
+                default_route[0]['gateway'], default_route[0]['interface-name']
```

Comment:
While we are at it, `keys` are always better then using `indexes` to refer items as the former is clearer, i would recommend we use a dict here

Reply:
See above. I will change that.

Reply:
Yup, something like the following would be nice:
```
{"ipv4": {"gateway": "", "interface": ""}, "ipv6": {"gateway": "", "interface": ""}}
```